tests/app/questionnaire/routing/test_operators.py

```diff
@@ -85,6 +97,15 @@ def test_operation_less_than(operands, expected_result):
     assert operator.evaluate(operands) is not expected_result
 
 
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    greater_than_and_less_than_operations_equals_test,
+)
+def test_operation_less_than_same_number(operands, expected_result):
+    operator = Operator(Operator.LESS_THAN)
+    assert operator.evaluate(operands) is expected_result
```

Comment:
This is already being testing and is not needed.

Reply:
See comment above